services/bettersocial/fcmToken/sendMultiDeviceNotification.js

```diff
@@ -15,7 +15,7 @@ const sendMultiDeviceNotification = async (
       const payload = {
         notification: {
           title: userNameFollower,
-          body: `${userNameFollower} just started following you. Say 'Hi'!`
+          body: `${userNameFollower} followed you. Say 'Hi'!`
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the notification message from `${userNameFollower} just started following you. Say 'Hi'!` to `${userNameFollower} followed you. Say 'Hi'!` is more concise and still conveys the same meaning.
```diff
- body: `${userNameFollower} just started following you. Say 'Hi'!`
+ body: `${userNameFollower} followed you. Say 'Hi'!`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->